summingbird-storm/src/main/scala/com/twitter/summingbird/storm/builder/FormattedScheme.scala

```diff
@@ -0,0 +1,27 @@
+package com.twitter.summingbird.storm.builder
+
+import com.twitter.tormenta.scheme.Scheme
+import java.nio.ByteBuffer
+import org.apache.storm.tuple.Fields
+import java.util.{ List => JList }
+import java.lang.{ Iterable => JIterable }
+import scala.collection.JavaConverters.asJavaIterableConverter
+
+/**
+ * This is custom wrapping `Scheme` class which is a way to customize Scheme's format.
+ */
+private[builder] sealed class FormattedScheme[T](scheme: Scheme[T], format: OutputFormat[T]) extends Scheme[T] {
+  override lazy val getOutputFields: Fields = format.asStormFields
+
+  override def deserialize(bytes: ByteBuffer): JIterable[JList[AnyRef]] =
+    try {
+      encode(decode(bytes))
+    } catch {
+      case t: Throwable => encode(handle(t))
+    }
+
+  private def encode(values: TraversableOnce[T]): JIterable[JList[AnyRef]] =
+    values.map(format.injection.apply).toIterable.asJava
+
+  override def decode(buf: ByteBuffer): TraversableOnce[T] = scheme.decode(buf)
```

Comment:
Because purpose of this class to actually override deserialize (which `storm`s method) to support `Format`.